docs/user-guides/configurations.md

```diff
@@ -72,6 +72,7 @@ Data-cleaning configurations are applicable to both Iceberg Format and Mixed str
 | snapshot.keep.duration                      | 720min(12 hours) | Table-Expiration keeps the latest snapshots within a specified duration                                                                                                                                                                                               |
 | snapshot.keep.min-count                     | 1                | Minimum number of snapshots retained for table expiration                                                                                                                                                                                                             |
 | clean-orphan-file.enabled                   | false            | Enables periodically clean orphan files                                                                                                                                                                                                                               |
+| clean-orphan-file.interval                  | 1d               | The interval for Cleaning orphan files within a specified time                                                                                                                                                                                                        |
```

Comment:
This change may need to be rollback

And you may want to add this new configuration example into `dist/src/main/amoro-bin/conf/config.yaml`